next.clayui.com/content/docs/components/popover.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Popover'
+description: 'Popovers are used together with the question mark icon to communicate to the user that there is useful information for them to read. Popovers contain helpful information such as an explanation of a context.'
```

Comment:
> are used together with the question mark icon

Since we don't actually include that functionality in the component, I think it might be best to leave that out. We can point to lexicon for best practices and use cases, but I dont think we should express what item to align it to.

Reply:
Makes sense :)